### DISCUSSION TITLE
Aerospace How to implement?  alt-right = 'move-node-to-monitor --wrap-around --focus-follows-window next' in aerospace.toml

### DISCUSSION THREAD

**Discussion by gaztec22** [nikitabobko/AeroSpace#661] [questions-and-answers] [answered]
Aerospace :

My intention is to use alt-right to move a window to the next monitor and focus on it

I'm using 
alt-right = 'move-node-to-monitor --wrap-around --focus-follows-window next

but I'm receiving an error unknown flag "--focus-follows-window"

What am I doing wrong here?

**Comment by gldtn** [comment]
It should be working fine, are you sure you're not mistyping anything?

I just tested on Aerospace 15.2-beta/Sequoia 15.1 and it's working for me with `alt-right = 'move-node-to-monitor --wrap-around --focus-follows-window next'`

At first I was getting an unknown flag because I missed an `s` on the `follows`.. If anything can you share your config so we can take a look? What Aerospace version are you running? What os version? Can you copy/paste the error back to us?

Hope it helps!

**Reply by gaztec22** [reply]
I've just updated to AeroSpace v0.15.2-Beta b6cf8277 / Ventura 13.7.1 and tried again using exactly:

alt-right = 'move-node-to-monitor --wrap-around --focus-folllows-window next'

When I save and tried to reload the config, I'm getting this error

Failed to parse /Users/<username>/.aerospace.toml

mode.main.binding.alt-right: ERROR: Unknown flag '--focus-folllows-window'
ERROR: Argument '(left|down|up|right|next|prev|<monitor-pattern>)' is mandatory

**Reply by Dimentium** [reply, marked as answer]
> I've just updated to AeroSpace v0.15.2-Beta b6cf8277 / Ventura 13.7.1 and tried again using exactly:
> 
> 
> 
> alt-right = 'move-node-to-monitor --wrap-around --focus-folllows-window next'
> 
> 
> 
> When I save and tried to reload the config, I'm getting this error
> 
> 
> 
> Failed to parse /Users/<username>/.aerospace.toml
> 
> 
> 
> mode.main.binding.alt-right: ERROR: Unknown flag '--focus-folllows-window'
> 
> ERROR: Argument '(left|down|up|right|next|prev|<monitor-pattern>)' is mandatory
> 
> 

You have a triple L in "follows"

**Reply by gaztec22** [reply]
oh yes! thanks for catching that.  Its working now :)
so I think the update to 15.2 beta and fixing the typo was the key. Thanks @gldtn  and @Dimentium  :)
